test/functionalTest/cases/3004_v2_reg_create/v2_reg_create.test

```diff
@@ -138,10 +138,13 @@ HTTP/1.1 200 OK
 Date: REGEX(.*)
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Content-Type: application/json
-Content-Length: 206
+Content-Length: 250
 
 {
     "counters": {
+        "deprecatedFeatures": {
```

Comment:
```suggestion
        "deprecatedRequests": {
```

Maybe the name of this field should be `deprecatedRequests` in order to align with the rest of the field?

Reply:
Discussed at Teams.

Although most of the counters are xxxxRequests, in this case, it seems better `deprecatedFeatures`.

NTC